src/ExpressiveInstaller/Resources/src/CallableFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Interop\Container\ContainerInterface;
```

Comment:
I'm commenting on the use statement for the typehint in the factory, not for the container being returned.

And PSR-11 ContainerInterface is a parent interface of the Interop one.